tests/distributed_mode/test_pipeline.py

```diff
@@ -63,3 +64,48 @@ def test_subpipeline_is_not_recreated_in_remote_workers():
     assert len(subpipelines) == 1
     assert len(documents) == 3
     assert len(nlp.pipeline[0].keys()) == 2
+
+
+def test_subpipeline_is_added_and_removed():
+
+    tagger = SimpleTagger(attribute="noun", lookups=["he", "she"], tag=True)
+
+    name = "my_tagger"
+    nlp.add_pipe(tagger, name=name)  # Add tagger to the subpipeline
+
+    # Assert nlp subpipeline has two components
+    assert len(nlp.pipeline) == 2  # Tokenizer is added by default
```

Comment:
This is the pipeline here, not the subpipeline.

Reply:
Done.

Reply:
Did u mean "Each dictionary might contain several pipes" here.
> Each dictionary might contain several subpipelines. 
Because the right next sentence contradicts it.

Reply:
Each dictionary might contain several keys each referring to a different subpipeline (But different in the sense that they are on different workers. But all contain the same chain of pipes)